deps/rabbit/src/rabbit_quorum_queue.erl

```diff
@@ -1528,35 +1555,35 @@ reclaim_memory(Vhost, QueueName) ->
     ra_log_wal:force_roll_over({?RA_WAL_NAME, Node}).
 
 %%----------------------------------------------------------------------------
-dead_letter_handler(Q, Overflow) ->
+dead_letter_handler(Q, Overflow, ShouldLog) ->
```

Comment:
I strongly dislike this extra argument and how it changes existing functions. If an invalid overflow strategy is used, I don't see a problem with logging that periodically.

Reply:
yes I agree, I'm not sure what value they add to this PR